ansible/ansible-ai-connect-service
# python 3.11

Jira Issue: https://issues.redhat.com/browse/AAP-20308

## Description
Upgrades to python 3.11 and latest ansible-lint as outlined in the issue.

## Testing
### Steps to test
1. Pull down the PR
2. Build and run the wisdom container locally. I struggled with this and bailed on it in favor of testing a PR deployment, so would be awesome if a reviwer who regularly works with containers can test this. See scenarios tested for what I did confirm.
3. ...

### Scenarios tested
Used 3.11 locally and pip installed everything and confirmed the service ran successfully run locally (not in a container).
Did a PR deployment and execed into the pod and confirmed:
sh-5.1$ /var/www/venv/bin/python3 --version
Python 3.11.5
Ran a completions request against the PR deployment and confirmed ansible lint was working (used a lowercase multi-task prompt and confirmed the first letter of the task name came back uppercase).

## Production deployment
